explorer/client/src/utils/api/DefaultRpcClient.ts

```diff
@@ -23,8 +23,15 @@ import type {
 
 export { Network, getEndpoint };
 
-export const DefaultRpcClient = (network: Network | string) =>
-    new JsonRpcProvider(getEndpoint(network));
+const defaultRpcMap: Map<Network | string, JsonRpcProvider> = new Map();
+export const DefaultRpcClient = (network: Network | string) => {
```

Comment:
I'm curious what the reason to keep these stored in the map is

Reply:
previously, we were re-creating them every time you asked for an rpc client.

this worked fine when using a stateless HTTP interface but it's an issue when using a stateful websocket, as it resulted in multiple connections being made to the node on a single explorer page instead of one